### PR TITLE
Remove all occurences of KubdeDB as it is not used nor supported anymore

### PR DESCRIPTION
**Pull request by KillianG**
Remove kubedb as it is not supported nor used anymore

Went through all integration branch, there is no kubedb occurence in other branchs as well

### CODE REVIEW

File: docs/docsource/installation/install/install_xdm.rst [hunk 1]
```diff
@@ -238,13 +235,11 @@ Deploy |product|
         version: |version|
         replicas: 1
         mongodb:
```
Comment (francoisferrand) [inline docs/docsource/installation/install/install_xdm.rst:252]: the whole `mongodb` field is not correct: in the current model, we deploy mongodb beforehand, and add the params of the "external" mongodb in the CR

File: solution/zenkoversion.yaml [hunk 2]
```diff
@@ -132,7 +132,6 @@ spec:
       exporter:
         image: redis_exporter
         tag: '${REDIS_EXPORTER_TAG}'
-      kubedb: '${REDIS_KUBEDB_TAG}'
```
Comment (francoisferrand) [inline solution/zenkoversion.yaml:135]: must also be done in the other zenkoversion.yaml

Comment (KillianG) [reply solution/zenkoversion.yaml:135]: What do you mean by "the other zenkoversion.yaml" ?

File: docs/docsource/installation/install/install_xdm.rst (hunk 1 above)
Comment (KillianG) [reply docs/docsource/installation/install/install_xdm.rst:252]: I am not sure I understand, should I just remove the mongodb filed ? If not, then what should it be exactly ?

Comment (francoisferrand) [reply docs/docsource/installation/install/install_xdm.rst:252]: I mean the `mongo` field configures Zenko for a specific mongo setup.

this doc was giving the whole kubedb config : changing the "provider" does not make the configuration correct, we need to update the doc with the current way to configure mongo in zenko cr (e.g. with external provider and with the other details)

you can check you artesca, you will see that the mongo field is not the same as all.

File: docs/docsource/installation/install/install_xdm.rst [hunk 3]
```diff
@@ -238,13 +235,11 @@ Deploy |product|
...
-          provider: KubeDB
           persistence:
             volumeClaimTemplate:
               size: 300Gi
               storageClassName: sc-300-g
         redis:
-          provider: KubeDB
```
Comment (francoisferrand) [inline docs/docsource/installation/install/install_xdm.rst:247]: same for redis, it is not about removing the provider: a provider still needs to be specified, and parameter adjusted accordingly.

File: solution/zenkoversion.yaml (hunk 2 above)
Comment (francoisferrand) [reply solution/zenkoversion.yaml:135]: there are 2 files named `zenkoversion.yaml` in the repo : the other one must be updated as well

File: docs/docsource/installation/install/install_xdm.rst (hunk 1 above)
Comment (KillianG) [reply docs/docsource/installation/install/install_xdm.rst:252]: I updated according to a real zenko in my platform
Needed to add a step to create the secret for mongodb credentials as it is required

**Review by francoisferrand — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by KillianG** [comment]
ping

**Comment by KillianG** [comment]
ping

**Pull request merged**